submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/service/filesystem/SubmissionFilesPermissions.kt

```diff
@@ -0,0 +1,16 @@
+package ac.uk.ebi.biostd.persistence.service.filesystem
+
+import java.nio.file.attribute.PosixFilePermission
+import java.nio.file.attribute.PosixFilePermissions
+
+enum class SubmissionFilesPermissions(val stringValue: String) {
+    ALL_CAN_READ("rwxr-xr-x"),
```

Comment:
This looks like a duplication of `RW_RW____` constants efforts, I mean the idea of those constants I understand is have al permissions string there, but now we also have then here. 

I am wondering if instead of constants maybe we need a sealed class with object (in case we need more features in them)